scripts/run_tempest.yml

```diff
@@ -13,23 +13,41 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Get defcore tests and set tempest_test_whitelist fact
+- name: Get defcore tests and set tempest_test_whitelist fact
+  hosts: utility_all[0]
+  gather_facts: no
+  user: root
+  tasks:
+    - include: get_defcore_tests.yml
```

Comment:
Why's an included set of tasks being used here? Is there intent to re-use those tasks somewhere? Why not just included them in this play?